flink-yarn/src/main/java/org/apache/flink/yarn/Utils.java

```diff
@@ -80,6 +80,9 @@
 	/** Yarn site xml file name populated in YARN container for secure IT run. */
 	public static final String YARN_SITE_FILE_NAME = "yarn-site.xml";
 
+	/** AM RM delegation token name populated in YARN container. */
+	public static final String AM_RM_TOKEN_NAME = "YARN_AM_RM_TOKEN";
```

Comment:
Where is it specified that the AM_RM token is called like this across all Yarn versions?

Reply:
The token kind is public evolving, and it has been like this since Hadoop 2.1 (see https://issues.apache.org/jira/browse/YARN-822).
 I've verified that it's still available on Hadoop 3.1 ( https://github.com/apache/hadoop/blob/55e0c134f002f74cb4a0360b6682a1b6796d1598/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/security/AMRMTokenIdentifier.java#L57).

Reply:
It occurs to me that we can use `AMRMTokenIdentifier.KIND_NAME` directly without introducing new dependencies since we're in `yarn-test` module.